client/modules/Run/components/RunList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Import Components
+import RunListItem from './RunListItem/RunListItem';
+
+function RunList(props) {
+  if (!props.runs) {
+    return (
+      <div className="emptyRunList">
+        <p>There are currently no runs in the DB.</p>
+      </div>);
+  // eslint-disable-next-line no-else-return
```

Comment:
looks like you're returning from else, so I don't think this is necessary?

Reply:
Naw. It doesn't like you following a return with an `else`. There's likely a better way to lay this out...but I couldn't get the syntax right for assigning jsx variables. Will take another stab at it though...just for you.

Reply:
Oh, that's because the else is redundant.  You can just delete it since you'll never get to the code after the if block when unless the if block is false.

Reply:
lmao. I'm an idiot. On the other hand, there is actually a better way. I'm testing it now.